docs/docs/charts/bar.mdx

```diff
@@ -174,6 +174,7 @@ The bar chart accepts the following configuration from the associated dataset op
 | `base` | `number` | | Base value for the bar in data units along the value axis. If not set, defaults to the value axis base value.
 | `maxBarThickness` | `number` | | Set this to ensure that bars are not sized thicker than this.
 | `minBarLength` | `number` | | Set this to ensure that bars have a minimum length in pixels.
+| [`pointStyle`](../configuration/elements.md#point-styles) | `string`\|`Image` | Yes | Yes | `'circle'`
```

Comment:
@LeeLenaleee sounds wrong table format. Default YES and Description YES.
If the option is scriptable, maybe it could be add in the Dataset Properties.

Reply:
yah coppied it to the wrong place